src/_igraph/graphobject.c

```diff
@@ -4832,7 +4832,7 @@ PyObject *igraphmodule_Graph_closeness(igraphmodule_GraphObject * self,
       return NULL;
     }
     if (igraph_closeness_cutoff(&self->g, &res, NULL, NULL, vs, mode, weights,
-        PyObject_IsTrue(normalized_o), PyFloat_AsDouble(cutoff_num))) {
+        PyFloat_AsDouble(cutoff_num), PyObject_IsTrue(normalized_o))) {
```

Comment:
The original order seems to be correct here (cf. `include/igraph_centrality.h` in the igraph/igraph repo) so I think this is not needed here.

Reply:
Sorry, I missed this when reviewing. You're right.